scripts/install-chef-suse.sh

```diff
@@ -344,20 +344,29 @@ EOF
     exit 1
 fi
 
-if ! (systemctl -q is-active crowbar-init.service && reset_crowbar ); then
-
+if ! (reset_crowbar 2> /dev/null); then
+  if systemctl --quiet is-active crowbar-init.service; then
```

Comment:
Travis is not happy due to indentation here (and on some other lines)

Reply:
I pushed a fixed version.